live-build/misc/ansible-roles/appliance-build.dcenter/tasks/main.yml

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- include: ldap.yml
+
+- apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - python-pip
+    - libldap2-dev
+    - libsasl2-dev
+    - isc-dhcp-server
+    - bind9
+    - dnsutils
+    - telnet
```

Comment:
I'm curious, what needs telnet?

Reply:
iirc, "dc console" telnets from the dcenter host to the esx host.

Reply:
Ah right, I forgot about that one!